tests/e2e/specs/in-game/one-offs/5_fives.spec.js

```diff
@@ -171,191 +167,11 @@ describe('FIVES', () => {
         cy.get('#deck').should('contain', '(1)');
       });
 
-      it('Draws only 1 card when last card in deck', () => {
```

Comment:
This is a necessary test as well. Generally we shouldn't be  removing tests unless the test itself is a duplicate or for whatever reason, the intended functionality is deliberately changing in such a way that the test is no longer relevant.

This is part of a rewrite where these tests will help us to verify that five resolution works correctly (and also that it doesn't get broken in the future)